copilot/fsd-authenticator/manifest.yml

```diff
@@ -120,6 +120,10 @@ environments:
   uat:
     http:
       alias: "authenticator.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk"
+      target_container: nginx
+      healthcheck:
+        path: /healthcheck
+        port: 8080
```

Comment:
should this change exist for dev and test too?

Reply:
it is optional but recommended.